packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-12/actions.mjs

```diff
@@ -0,0 +1,133 @@
+import { promises as fs } from 'fs';
+import * as path from 'path';
+import { dirname as dname } from 'path';
+import { fileURLToPath } from 'url';
+import { voteLatestProposalAndWait } from '../coreUpgradeHelpers.mjs';
+import { CHAINID, GOV1ADDR, VALIDATORADDR } from '../constants.mjs';
+import { agd, bundleSource } from '../cliHelper.mjs';
+
+const dirname = dname(fileURLToPath(import.meta.url));
+
+export const installBundles = async bundlesData => {
```

Comment:
this seems useful in more than one actions file. consider putting in `coreUpgradeHelpers`